compiler/wrap.js

```diff
@@ -188,7 +221,10 @@ export default (source, flags = [ 'module' ], customImports = {}, print = str =>
       if (!mismatch) break;
     }
 
-    if (i === wasm.length) return false;
+    if (i === wasm.length) {
+      printDecomp(-1, func, funcs, globals, exceptions)
```

Comment:
nit: no `;` here and on a few other lines

Reply:
Fixed, at least the ones I found, pretty sure I caught them all.